rendered/draft-nuttycom-lockbox-streams.html

```diff
@@ -1,113 +1,3 @@
-                        <span class="math">\(\mathsf{PoolValue}_{Deferred}\)</span>
-                    .</p>
-                </blockquote>
-                <p>The protocol specification is modified to define the "total issued supply" such that the total issued supply as of a given height is given by the function:</p>
-                <div class="math">\(\begin{array}{ll}
-\mathsf{IssuedSupply}(\mathsf{height}) := &amp;\!\!\!\!\mathsf{PoolValue}_{Transparent}(\mathsf{height}) \\
-&amp;+\;\; \mathsf{PoolValue}_{Sprout}(\mathsf{height}) \\
-&amp;+\,\; \mathsf{PoolValue}_{Sapling}(\mathsf{height}) \\
-&amp;+\,\; \mathsf{PoolValue}_{Orchard}(\mathsf{height}) \\
-&amp;+\,\; \mathsf{PoolValue}_{Deferred}(\mathsf{height})
-\end{array}\)</div>
-            </section>
-        </section>
-        <section id="references"><h2><span class="section-heading">References</span><span class="section-anchor"> <a rel="bookmark" href="#references"><img width="24" height="24" class="section-anchor" src="assets/images/section-anchor.png" alt=""></a></span></h2>
-            <table id="bcp14" class="footnote">
-                <tbody>
-                    <tr>
-                        <th>1</th>
-                        <td><a href="https://www.rfc-editor.org/info/bcp14">Information on BCP 14 — "RFC 2119: Key words for use in RFCs to Indicate Requirement Levels" and "RFC 8174: Ambiguity of Uppercase vs Lowercase in RFC 2119 Key Words"</a></td>
-                    </tr>
-                </tbody>
-            </table>
-            <table id="zip-1014" class="footnote">
-                <tbody>
-                    <tr>
-                        <th>2</th>
-                        <td><a href="zip-1014">ZIP 1014: Establishing a Dev Fund for ECC, ZF, and Major Grants</a></td>
-                    </tr>
-                </tbody>
-            </table>
-            <table id="zip-0207" class="footnote">
-                <tbody>
-                    <tr>
-                        <th>3</th>
-                        <td><a href="zip-0207">ZIP 207: Funding Streams</a></td>
-                    </tr>
-                </tbody>
-            </table>
-            <table id="zip-0207-funding-streams" class="footnote">
-                <tbody>
-                    <tr>
-                        <th>4</th>
-                        <td><a href="zip-0207#funding-streams">ZIP 207: Funding Streams. Section: Funding streams</a></td>
-                    </tr>
-                </tbody>
-            </table>
-            <table id="zip-0207-consensus-rules" class="footnote">
-                <tbody>
-                    <tr>
-                        <th>5</th>
-                        <td><a href="zip-0207#consensus-rules">ZIP 207: Funding Streams. Section: Consensus rules</a></td>
-                    </tr>
-                </tbody>
-            </table>
-        </section>
-    </section>
-</body>
-</html>
\ No newline at end of file
+---
+redirect_to: "https://zips.z.cash/zip-2001"
+---
```

Comment:
I don't think this will work. GitHub pages will only serve a static site. The sites that use Jinja have a GitHub Action that renders the template to HTML; here you're trying to include the template in the HTML. I think this will work:
```suggestion
<!DOCTYPE html>
<html>
<head>
    <title>ZIP 2001: Lockbox Funding Streams (redirect)</title>
    <meta http-equiv="refresh" content="0; https://zips.z.cash/zip-2001">
</head>
</html>
```

Reply:
It was unclear to me how GitHub Pages Jinja was configured. The existence of `_config.yml` in the repo implied that you were already using it, and AFAICT the way you configure Jinja for a given page is to prepend YML to the HTML.

That being said, indeed this particular gem is intended to generate the equivalent of the HTML you've given, so if the Jinja stuff doesn't work, then we should a) use your suggestion, and b) delete `_config.yml` to stop it being misleading.

Reply:
I think `_config.yml` is probably a leftover, but I could be wrong.

Reply:
We will find out one way or another once this PR merges.